### DISCUSSION TITLE
Allow transparent backgrounds for PDF rendering

### DISCUSSION THREAD

**Discussion by Ugoku** [spatie/browsershot#628] [Ideas]
We would like to have a PDF rendered with a transparent background, so we can combine this with a different PDF background using pdftk. Puppeteer can do this, using `omitBackground`.

Can you please add this as an option, or do it by default? Not sure if there are drawbacks to doing it by default. I'm willing to create a PR if you prefer.

Bedankt!

**Comment by freekmurze** [comment]
I'd accept a PR for this, make sure to update the readme and tests 👍
